crates/biome_package/src/node_semver/version.rs

```diff
@@ -26,11 +26,11 @@ use std::str::FromStr;
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Version {
     /// The major version number. Incremented for incompatible API changes.
-    pub major: u64,
+    pub major: u16,
     /// The minor version number. Incremented for backwards-compatible functionality additions.
-    pub minor: u64,
+    pub minor: u16,
     /// The patch version number. Incremented for backwards-compatible bug fixes.
-    pub patch: u64,
+    pub patch: u16,
     /// Pre-release version identifiers (e.g., ("alpha", "1") for "1.2.3-alpha.1").
     pub prerelease: Vec<String>,
```

Comment:
Because `prerelease` and `build` are rarely used, I wonder if this could be beneficial of adding a single field `prerelease_build: Box<str>`. `Prerelease` and `build` are already semi-parsed data. This could make them unparsed.